disperser/cmd/encoder/main.go

```diff
@@ -49,16 +51,52 @@ func RunEncoderServer(ctx *cli.Context) error {
 		return err
 	}
 
-	enc, err := NewEncoderGRPCServer(config, logger)
-	if err != nil {
-		return err
+	metrics := encoder.NewMetrics(config.MetricsConfig.HTTPPort, logger)
+	if config.MetricsConfig.EnableMetrics {
+		httpSocket := fmt.Sprintf(":%s", config.MetricsConfig.HTTPPort)
+		metrics.Start(context.Background())
+		logger.Info("Enabled metrics for Encoder", "socket", httpSocket)
 	}
-	defer enc.Close()
 
-	err = enc.Start(context.Background())
+	if config.EncoderVersion == V2 {
+		// We no longer compute the commitments in the encoder, so we don't need to load the G2 points
+		prover, err := prover.NewProver(&config.EncoderConfig, false)
+		if err != nil {
+			return fmt.Errorf("failed to create encoder: %w", err)
+		}
+
+		s3Client, err := s3.NewClient(context.Background(), config.AwsClientConfig, logger)
+		if err != nil {
+			return err
+		}
+
+		blobStoreBucketName := config.BlobStoreConfig.BucketName
+		blobStore := blobstorev2.NewBlobStore(blobStoreBucketName, s3Client, logger)
+		logger.Info("Blob store", "bucket", blobStoreBucketName)
+
+		chunkStoreBucketName := config.ChunkStoreConfig.BucketName
+		chunkWriter := chunkstore.NewChunkWriter(logger, s3Client, chunkStoreBucketName, 4*1024*1024) // Unlikely to change so we choose to hardcode the fragment size at 4Mb
```

Comment:
Pull this literal numeric out as a constant? Having a name makes it a bit more clear what this parameter is.